### PR TITLE
Add mention of the karma threshold for downvoting

### PR DESCRIPTION
**Pull request by yaymukund**
I added a bit in the about page about needing 50 karma to downvote. 😊

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -97,6 +97,7 @@
 
   <h2 id="downvotes">Downvote Explanations</h2>
   <p>
+  Users may downvote stories and comments after their account has 50 karma.
```
Comment (Carpetsmoker) [inline app/views/home/about.html.erb]: This should probably use the `User.MIN_KARMA_TO_DOWNVOTE` constant.

Comment (yaymukund) [reply app/views/home/about.html.erb]: I've done the suggestion you suggested

**Comment by yaymukund** [comment]
done!

**Comment by pushcx** [comment]
Thanks for adding this. Could you move the sentence to the end of the paragraph? It reads better to introduce the overall system and then talk about the corner cases at the end. Thanks!

**Comment by pushcx** [comment]
Thanks for this addition.

**Pull request merged**